### ISSUE TITLE
pip install doesn't work

### ISSUE THREAD

**Issue by SergeyPirogov** [reportportal/agent-python-pytest#3]
Tried to install using pip install got an error:
```
Collecting pytest-reportportal
  Could not find a version that satisfies the requirement pytest-reportportal (from versions: )
No matching distribution found for pytest-reportportal
```

**Comment by krasoffski** [comment]
Hi,
Could you provide initial command?
It seem like there is no pytest-reportportal in the pypi.python.org repository.

**Comment by ppapou** [comment]
Hello,
The version 0.2.3 with is added to the PyPi.

**Comment by DzmitryHumianiuk** [comment]
fixed within 0.2.3

**Comment by SergeyPirogov** [comment]
Yeeees https://pypi.python.org/pypi/pytest-reportportal. Thanks for quick fix
